lambdas/src/Server.ts

```diff
@@ -41,7 +41,26 @@ export class Server {
       this.app.use(compression({ filter: (req, res) => true }))
     }
 
-    this.app.use(cors())
+    const corsOptions: cors.CorsOptions = {
+      origin: true,
+      methods: 'GET,HEAD,POST,PUT,DELETE,CONNECT,OPTIONS,TRACE,PATCH',
+      allowedHeaders: [
+        'DNT',
+        'X-Mx-ReqToken',
+        'Keep-Alive',
```

Comment:
This applies to all configs: why do we need those headers with cors?

Reply:
I wasn't sure which headers do the client send, and if they are sending a non-allowed Header, it will fail. So, I took a list from common headers and added `X-Peer-Token`

Reply:
I'd say we focus on the headers we are interested-in, ignoring what the client actually sends. Otherwise when anyone come to this code in the future would have to research what does `X-Mx-ReqToken` is and why it is there. And it is another thing we will have to maintain indefinitely